unittest.sh

```diff
@@ -14,3 +14,4 @@ if [ -n "$DEBUG" ] ; then
 fi
 
 /usr/bin/php support/unittester/run-saffire-tests.php ${1:-"unittests/tests/"}
+./build/release/utmain
```

Comment:
Would it make more sense to start with the core tests first? (ie: swap these two lines?)